<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettingsDialogClass</class>
 <widget class="QDialog" name="EditorSettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑器设置</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::InputMethodHint::ImhNone</set>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>56</width>
     <height>114</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="fontPage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>56</width>
      <height>54</height>
     </rect>
    </property>
    <attribute name="label">
     <string>字体</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="colorsPage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>56</width>
      <height>54</height>
     </rect>
    </property>
    <attribute name="label">
     <string>配色方案</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="previewWidget" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="previewLabel">
      <property name="text">
       <string>效果预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="editorWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="buttonWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>831</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>567</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="confirmPushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyPushButton">
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="settingsFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>341</width>
     <height>203</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="fontFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="fontLabel">
           <property name="text">
            <string>字体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="fontSizeLabel">
           <property name="text">
            <string>大小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fontSizeLineEdit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="colorFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="backGroundColorLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="backGroundColorLabel">
         <property name="text">
          <string>背景颜色</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="backGroundColorPushButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none; 
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../resource.qrc">
           <normaloff>:/icon/resources/icons/color_board.svg</normaloff>:/icon/resources/icons/color_board.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="backGroundColorFrame">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background:#2E2E2E
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="foreGroundColorLabel">
         <property name="text">
          <string>前景颜色</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
